ts-types/iterable/ensure.d.ts

```diff
@@ -0,0 +1,7 @@
+import { EnsureFunction, EnsureOptions } from '../ensure';
+
+type IterableEnsureOptions = { ensureItem?: EnsureFunction, allowString?: boolean, denyEmpty?: boolean} | EnsureOptions;
+
+declare function ensureIterable<T>(value: any, options?: IterableEnsureOptions): T[];
+declare function ensureIterable(value: any, options?: IterableEnsureOptions): any[];
```

Comment:
@medikoo do we want to clean up this one as well?

Reply:
Yes, that'll be great. still, it shares same caveat, that it's not necessary input object that'll be returned. e.g. input can be `Set`, and with `esnureItem` output will be `Array`